tests/manifests/test-kms-vault-spec-k8s-auth.yaml

```diff
@@ -12,3 +12,6 @@ spec:
         VAULT_CACERT: "vault-ca-cert"
         VAULT_AUTH_METHOD: kubernetes
         VAULT_AUTH_KUBERNETES_ROLE: "rook-ceph"
+    keyRotation:
```

Comment:
the key rotation was needed for CI to pass?

Reply:
Since the patch was causing some error I had add it here.